content/advent-2018/observability-in-the-kitchen.md

```diff
@@ -0,0 +1,200 @@
++++
+author = ["Daisy Tsang"]
+title = "Improve Your Breadmaking Skills with Go and Open-Source Monitoring"
+linktitle = "Observability in the Kitchen"
+date = "2018-12-11T00:00:00Z"
+series = ["Advent 2018"]
++++
+
+
+I have many different interests, including baking, open-source software, and more recently, systems monitoring and learning Go.  As a way for me to expand my practical knowledge on each item, I devised a fun little project that leverages sensors, Raspberry Pis, and [Prometheus](https://prometheus.io) to improve my sourdough breadmaking process. I will explain the inspiration behind it, go through the steps I took to set up the project, and detail what I learned.  Disclaimer: it is still a work in progress!
+
+
+### Sourdough Bread and the Difficulty with Maintaining Natural Yeast
+A while ago, I became interested in the art of making one’s own sourdough bread.  This is a type of bread that, due to its fermentation process, is a much healthier alternative to the store bought bread that we are used to seeing. The naturally occurring acids and long fermentation help to break down the proteins and gluten, making it more digestible and easy for the body to absorb. 
+
+Sourdough is the traditional way of making bread until around a hundred years ago.  The process involves cultivating your own wild yeast and allowing time for fermentation.  The sourdough bread-making process is a fascinating combination of biology and physics, but working with natural starters can be difficult since they are very sensitive to temperature and humidity. The cultures are very funnicky and can easily turn to mold or not grow as fast as you would like. Parameters such as temperature and humidity need to be closely observed and monitored...perhaps with a systems monitoring tool!
```

Comment:
funnicky -> finicky ??

Reply:
Yess I meant finicky, oops!

Reply:
I fixed the typo, changed some wordings, and added some graphs!